manifests/baremetal/keepalived.yaml

```diff
@@ -49,26 +49,50 @@ spec:
         if pid=$(pgrep -o keepalived); then
             kill -s SIGHUP "$pid"
         else
-            /usr/sbin/keepalived -f /etc/keepalived/keepalived.conf --dont-fork --vrrp --log-detail --log-console &
+            start_keepalived
         fi
       }
+
+      stop_keepalived()
+      {
+        if pid=$(pgrep -o keepalived); then
+          kill "$pid"
```

Comment:
Two suggestions:

- Be specific about the signal we are sending
- Wait for the graceful termination.

Reply:
Do you mean add a kill -9 if it doesn't terminate gracefully (which seems like a good thing to do)? Otherwise there isn't much need to wait since we aren't going to do anything else after it exits.

Reply:
Yeah, I meant wait a reasonable time for SIGTERM to do its thing (probably we can use the same systemd uses) and then SIGKILL it.

Reply:
Yeah, that's possible. I'll make the change.

Reply:
Okay, this should be done. I set the timeout to 9 seconds since the monitor runs every 10 and I doubt we want to have multiple stop commands going at once.